### DISCUSSION TITLE
Add Research Papers

### DISCUSSION THREAD

**Discussion by GabinL21** [dailydotdev/daily#1672] [Feature Requests]
Hi! I've been using daily.dev for more than a year and I love it!

It replaced every other thing I used for technical watch, besides one thing: Google Scholar for research papers. 

I'm not sure how big of an effort it would be (between really big and near impossible I guess), but I think it would be really cool. They are often reliable source of knowledge, and could greatly benefit for your blog post features (tags, feeds, etc.) It would be even better with the AI resume, to quickly get the main info out of sometimes hard to understand research papers. This is DEFINITELY a feature I would pay for.

**Comment by idoshamun** [comment]
Hi @GabinL21,

We had this suggestion in the past but never had the time to look into it.
I'd love to hear more, specifically about your workflow and how you discover and read research papers today.
